x-pack/plugins/index_management/public/application/components/component_templates/component_template_selector/components/filter_list_button.tsx

```diff
@@ -76,7 +83,10 @@ export function FilterListButton({ onChange, filters }: Props) {
         panelPaddingSize="none"
         data-test-subj="filterList"
       >
-        <div className="euiFilterSelect__items">
+        {/* EUI NOTE: Please use EuiSelectable (which already has height/scrolling built in)
+            instead of EuiFilterSelectItem (which is pending deprecation).
+            @see https://elastic.github.io/eui/#/forms/filter-group#multi-select */}
+        <div className="eui-yScroll" css={{ maxHeight: euiTheme.base * 30 }}>
```

Comment:
I think there is a small regression for this one, let me know if you need help fixing.

// Before
<img width="586" alt="Screenshot 2023-07-31 at 11 47 20 AM" src="https://github.com/elastic/kibana/assets/5226211/761a1ab6-888b-4dd9-b455-02e7a4ec080c">

// After
<img width="596" alt="Screenshot 2023-07-31 at 11 48 26 AM" src="https://github.com/elastic/kibana/assets/5226211/992ec42f-d8ee-44ee-9dc2-b98e796b0fa0">

Reply:
The above code doesn't actually affect the button that triggers the popover, only the popover itself. If this is broken, it would likely have been caused by `EuiFilterButton`'s Emotion conversion which was in a previous PR. It may also be related to https://github.com/elastic/kibana/issues/161464 which should have been fixed by https://github.com/elastic/kibana/pull/161592 🤔 

I'm not totally sure where to repro the above screenshots - can you provide a local link?

Reply:
@alisonelizabeth This should be fixed with https://github.com/elastic/kibana/pull/162660/commits/d3040e35047ede89d6265cfe5be0c6829aef3035 - please let me know if not!

Reply:
Thank you for fixing! Looks good now.